main.py

```diff
@@ -1,42 +1,128 @@
+id_user = 0
+balance = 100
+state = None
+list_of_users = []
```

Comment:
it'd be way better to not use 2d array but 1d array of objects of class User, which you could've created